Cork/Views/Settings/Settings View.swift

```diff
@@ -27,7 +27,7 @@ struct SettingsView: View
 
             BrewPane()
                 .tabItem {
-                    Label("Brew", systemImage: "cup.and.saucer")
+                    Label("Homebrew", systemImage: "cup.and.saucer")
```

Comment:
Refer to Homebrew consistently as 'Homebrew' instead of 'Brew' – seemed clearer to me

Reply:
Agreed